### PR TITLE
Reclaim pv in 'released' state and put them back in kubernetes

### PR DESCRIPTION
**Pull request by Zempashi**

### CODE REVIEW

File: roles/reclaim_local_storage/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  shell: rm -rf {{ item.spec.local.path }}/*
+  args:
+    warn: False
+  loop_control:
+     label: "{{ inventory_hostname }}-{{ item.spec.local.path }}"
+  with_items: '{{ released_pv[inventory_hostname]|default([]) }}'
+
+- name: 'delete pv'
+  uri:
+    url: 'http://localhost:8080/api/v1/persistentvolumes/{{ item.metadata.name }}'
```
Comment (NicolasT) [inline roles/reclaim_local_storage/tasks/main.yml]: How is auth handled?

Comment (Zempashi) [reply roles/reclaim_local_storage/tasks/main.yml]: No auth, talk to (unauthenticated) localhost for now

File: roles/reclaim_local_storage/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    method: 'DELETE'
+  run_once: True
+  delegate_to: '{{ groups["kube-master"]|first }}'
+  loop_control:
+     label: "{{ item.metadata.name }}"
+  with_items: '{{ released_pv.values()|list }}'
+
+- name: 'recreate pv'
```
[the remark this answers is not in the document]
Comment (Zempashi) [reply roles/reclaim_local_storage/tasks/main.yml]: Could be. Wanted a fast playbook, that can work on any pre-deployed (no need to adjust group vars)

File: roles/reclaim_local_storage/tasks/main.yml (hunk 1 above)
Comment (NicolasT) [reply roles/reclaim_local_storage/tasks/main.yml]: Good point.

**Pull request merged**